hydromt_fiat/api/data_types.py

```diff
@@ -105,18 +105,18 @@ class ExposureSetupDamages(BaseModel):
     max_dist: Optional[Union[float, int, List[float], List[int]]] = None
     damage_types: Optional[Union[str, List[str]]] = None
 
-class RoadVulnerabilitySettings(BaseModel):
-    threshold_value: float
-    min_hazard_value: float
-    max_hazard_value: float
-    step_hazard_value: float
-    vertical_unit: Units
+#class RoadVulnerabilitySettings(BaseModel):
+    #threshold_value: float
+    #min_hazard_value: float
+    #max_hazard_value: float
+    #step_hazard_value: float
+    #vertical_unit: Units
```

Comment:
why did you need to comment this out? I was expecting that you will only comment out only the part that uses the method to setup the roads (set_road_damage_threshold). 

But I don't see any problem for the actual classes and functions to still be there.